rotors_gazebo/resource/fixed_wing/techpod.yaml

```diff
@@ -12,22 +12,22 @@ throttle_channel: 5
 #   [rad] Maximum deflection angle
 #   Channel number in the array of normalized actuator commands
 aileron_left:
-  channel: 4
+  channel: 0
```

Comment:
should `aileron_left` and `aileron_right` be on the same channel?
probably @hitimo you should take a look at this :)